rum/connection/database.py

```diff
@@ -3,39 +3,29 @@
 
 
 class Redshift(object):
-    _instance = None
-
-    def __new__(cls):
-        if cls._instance is None:
-            cls._instance = object.__new__(cls)
-
-            try:
-                db_config = {'dbname': os.environ['DBNAME'],
-                             'host': os.environ['HOST'],
-                             'password': os.environ['PWD'],
-                             'port': os.environ['PORT'],
-                             'user': os.environ['USER']}
-            except KeyError as error:
-                Redshift._instance = None
-                raise KeyError('Error: Please set connection credentials as environmental vars. {}'.format(error))
-
-            try:
-                print('connecting to Redshift database...')
-                connection = Redshift._instance.connection = psycopg2.connect(**db_config)
-                cursor = Redshift._instance.cursor = connection.cursor()
-                cursor.execute('SELECT VERSION()')
-                db_version = cursor.fetchone()
-            except Exception as error:
-                Redshift._instance = None
-                raise ConnectionError('Error: connection not established {}'.format(error))
-            else:
-                print('connection established\n{}'.format(db_version[0]))
-
-        return cls._instance
-
     def __init__(self):
-            self.connection = self._instance.connection
-            self.cursor = self._instance.cursor
+        # Config DB credentials
+        db_config = {'dbname': os.environ.get('DBNAME', 'database'),
+                     'host': os.environ.get('HOST', 'localhost'),
+                     'password': os.environ.get('PASSWORD', ''),
+                     'port': os.environ.get('PORT', '5439'),
+                     'user': os.environ.get('USER', 'user')}
+
+        # Connect to DB
+        try:
```

Comment:
You can extract this whole block to a method like `create_connection` or `stablish_connection`.